pkg/resources/management.cattle.io/v3/projectroletemplatebinding/validator.go

```diff
@@ -91,16 +96,19 @@ func (v *Validator) Admit(request *admission.Request) (*admissionv1.AdmissionRes
 	}
 
 	if request.Operation == admissionv1.Create {
-		if err = v.validateCreateFields(prtb); err != nil {
-			return &admissionv1.AdmissionResponse{
-				Result: &metav1.Status{
-					Status:  "Failure",
-					Message: err.Error(),
-					Reason:  metav1.StatusReasonBadRequest,
-					Code:    http.StatusBadRequest,
-				},
-				Allowed: false,
-			}, nil
+		if err := v.validateCreateFields(prtb, fieldPath); err != nil {
+			if errors.As(err, &fieldErr) {
+				return &admissionv1.AdmissionResponse{
+					Result: &metav1.Status{
+						Status:  "Failure",
+						Message: err.Error(),
```

Comment:
I think this should be
```suggestion
						Message: fieldErr.Error(),
```

Reply:
either works in the current form `err` would be more verbose if the field.Error was wrapped.